test/framework/envs/universal/env.go

```diff
@@ -89,5 +77,8 @@ func AfterSuite(report ginkgo.Report) {
 	if framework.Config.CleanupLogsOnSuccess {
 		universal_logs.CleanupIfSuccess(framework.Config.UniversalE2ELogsPath, report)
 	}
-	PrintCPLogsOnFailure(report)
+	if !report.SuiteSucceeded {
+		framework.Logf("Please see full CP logs by downloading the debug artifacts")
```

Comment:
I guess we can't create a link?

Reply:
It's not easy to create the link without importing the GH SDK, or by referencing some environment variables which indedicate the run id and repo name, etc.